src/api/Cargo.toml

```diff
@@ -22,8 +22,8 @@ actix-web-httpauth = "0.8.0"
 aes-gcm = "0.10.1"
 anyhow.workspace = true
 async-trait.workspace = true
-as-types = { git = "https://github.com/confidential-containers/attestation-service.git", tag = "v0.7.0"}
-attestation-service = { git = "https://github.com/confidential-containers/attestation-service.git", default-features = false, tag = "v0.7.0", optional = true}
+as-types = { git = "https://github.com/chendave/attestation-service.git", branch = "fvp"}
+attestation-service = { git = "https://github.com/chendave/attestation-service.git", branch = "fvp", default-features = false, optional = true}
```

Comment:
those change should be updated once the change in attestation-service is in and tag is created.